miniapp/src/main/AndroidManifest.xml

```diff
@@ -30,6 +30,15 @@
                 android:name="android.support.FILE_PROVIDER_PATHS"
                 android:resource="@xml/miniapp_downloaded_file_provider_paths" />
         </provider>
+        <provider
+            android:name="androidx.core.content.FileProvider"
```

Comment:
The default FileProvider shouldn't be used because it could conflict with the Host App if they are also using the default `androidx.core.content.FileProvider`.

I think you could re-use the `com.rakuten.tech.mobile.miniapp.display.MiniAppDownloadedFileProvider` from above instead of creating a new FileProvider. It should already be the same as the default. Maybe it just needs to be renamed to something like `MiniAppDefaultFileProvider`?

Reply:
I was trying to re-use `com.rakuten.tech.mobile.miniapp.display.MiniAppDownloadedFileProvider`, unfortunately it was not opening the camera :( 
Let's see if @munir-rakuten found something.